src/pages/settings/expense-categories/components/CreateExpenseCategoryModal.tsx

```diff
@@ -27,7 +27,6 @@ export function CreateExpenseCategoryModal(props: Props) {
       title={t('create_expense_category')}
       visible={props.visible}
       onClose={() => props.setVisible(false)}
-      size="small"
     >
```

Comment:
The reason for removing this size is because we now have an `extraSmall` and `small` size. Now the small is a bit wider than before, so I had to remove it to set the `extraSmall' size as the default.